[32mINFO    [0m BrowserLogger:conftest.py:54 	Test "test_create_new_user" is running
[32mINFO    [0m BrowserLogger:conftest.py:115 Browser chrome started with such capabilities: {'acceptInsecureCerts': False, 'browserName': 'chrome', 'browserVersion': '96.0.4664.110', 'chrome': {'chromedriverVersion': '96.0.4664.45 (76e4c1bb2ab4671b8beba3444e61c0f17584b2fc-refs/branch-heads/4664@{#947})', 'userDataDir': 'C:\\Users\\MAF23~1.ELI\\AppData\\Local\\Temp\\scoped_dir2096_1731441488'}, 'goog:chromeOptions': {'debuggerAddress': 'localhost:59972'}, 'networkConnectionEnabled': False, 'pageLoadStrategy': 'normal', 'platformName': 'windows', 'proxy': {}, 'setWindowRect': True, 'strictFileInteractability': False, 'timeouts': {'implicit': 0, 'pageLoad': 300000, 'script': 30000}, 'unhandledPromptBehavior': 'dismiss and notify', 'webauthn:extension:credBlob': True, 'webauthn:extension:largeBlob': True, 'webauthn:virtualAuthenticators': True}
[32mINFO    [0m UserLoginPage:UserLoginPage.py:17 Opening source: http://10.0.2.15/index.php?route=account/register
[32mINFO    [0m BrowserLogger:conftest.py:119 	Test "test_create_new_user" completed